src/client/terrain/util/blocks.rs

```diff
@@ -1,8 +1,58 @@
 use crate::prelude::*;
 
+#[derive(Hash, Eq, PartialEq, Copy, Clone)]
+pub enum TextureName {
+    Air,
+    Stone,
+    Dirt,
+    GrassTop,
+    GrassSide,
+    Bedrock,
+    RedSand,
+    BrownTerracotta,
+    CyanTerracotta,
+    GrayTerracotta,
+    LightGrayTerracotta,
+    OrangeTerracotta,
+    RedTerracotta,
+    Terracotta,
+    YellowTerracotta,
+}
+
+#[derive(Resource)]
+pub struct TextureManager {
+    textures: HashMap<TextureName, TextureUV>,
+}
+
+impl TextureManager {
+    pub fn new() -> Self {
```

Comment:
TODO: Introduce cfg files for texture coordinates / dynamic texture atlas generation from texture names